pyposeidon/boundary.py

```diff
@@ -22,11 +22,11 @@ def simplify(geo):
     geo = geo.loc[:, ["geometry"]].explode(index_parts=True).droplevel(0).reset_index(drop=True)
 
     # Just in case merge
```

Comment:
If this is something that normally shouldn't happen, perhaps adding a log message would not be a bad idea. Might help identify some issue

Reply:
This relates to the quality of the input coastlines. This is a function to  check/fix any inconsistencies but it is not bulletproof. Generally speaking, it is the user who should provide a quality coastlines dataset.